docs/reference/concepts/02-provider.mdx

```diff
@@ -80,12 +80,42 @@ However, the following items should be used in the package name:
 
 An example of what it could look like in NPM: `@openfeature/flagd-provider`
 
-#### Multi-provider support
+### Domain scopped providers
 
-OpenFeature supports multiple providers in a single application, allowing teams to scope flag evaluation to a particular flag management system.
-This makes it easier for developers to mix and match providers based on their needs, all from a unified SDK.
+OpenFeature supports multiple providers [domain-scoped](https://openfeature.dev/specification/glossary/#domain) to multiple clients in a single application.
+This allows teams to scope flag evaluation to a particular flag management system, making it easier for developers to mix and match providers based on their needs, all from a unified SDK.
 
-> To learn more, please refer to the [OpenFeature Enhancement Proposal](https://github.com/open-feature/ofep/blob/main/OFEP-provider-client-mapping.md) or [spec change](https://openfeature.dev/specification/sections/flag-evaluation/#requirement-113).
+```typescript
+OpenFeature.setProvider('ProviderA', new ProviderA());
+OpenFeature.setProvider('ProviderB', new ProviderB());
+
+const clientA = OpenFeature.getClient('ProviderA');
+const clientB = OpenFeature.getClient('ProviderB');
+```
+
+> To learn more, please refer to the [OpenFeature Enhancement Proposal](https://github.com/open-feature/ofep/blob/main/OFEP/provider-client-mapping.md) or [spec change](https://openfeature.dev/specification/sections/flag-evaluation/#requirement-113).
```

Comment:
I wonder if we should move the OFEP files to the docs and archive the repo. 🤔

Reply:
That's probably a good idea

Reply:
I'll resolve this thread but it's something we may want to address in the future.